maintainers/scripts/feature-freeze-teams.pl

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env nix-shell
```

Comment:
Ugh, perl. Can we write this in something more approachable, like python, so people feel empowered to change this script?

I think this was one of the main reasons on why we switched to using python for the VM test framework.

Reply:
Feel free